app/views/handlers/confirm_email.py

```diff
@@ -153,17 +148,17 @@ def _publish_fulfilment_request(self):
 @dataclass
 class ConfirmationEmailFulfilmentRequest(FulfilmentRequest):
     email_address: str
-    session_data: SessionData
+    questionnaire_store: QuestionnaireStore
```

Comment:
We should only be passing info that is needed. Here, it should be passing in metadata, not the entire questionnaire store. Technically it should pass the 3 items it needs not the entire metadata either but happy to keep that given that is what we were doing with session data.